documentation/src/main/asciidoc/mapstruct-reference-guide.asciidoc

```diff
@@ -350,14 +357,31 @@ public class CarMapperImpl implements CarMapper {
 ----
 ====
 
-The general philosophy of MapStruct is to generate code which looks as much as possible as if you had written it yourself from hand. In particular this means that the values are copied from source to target by plain getter/setter invocations instead of reflection or similar.
+The general philosophy of MapStruct is to generate code which looks as much as possible as if you had written it yourself from hand. In particular this means that the values are copied from source to target by plain getter/setter/field invocations instead of reflection or similar.
 
 As the example shows the generated code takes into account any name mappings specified via `@Mapping`. If the type of a mapped attribute is different in source and target entity, MapStruct will either apply an automatic conversion (as e.g. for the _price_ property, see also <<implicit-type-conversions>>) or optionally invoke another mapping method (as e.g. for the _driver_ property, see also <<mapping-object-references>>).
 
 Collection-typed attributes with the same element type will be copied by creating a new instance of the target collection type containing the elements from the source property. For collection-typed attributes with different element types each element will be mapped individually and added to the target collection (see <<mapping-collections>>).
 
 MapStruct takes all public properties of the source and target types into account. This includes properties declared on super-types.
 
+[[field-mappings]]
```

Comment:
Let's put this section to the end of chapter 3 (below chapter `[[updating-bean-instances]]`. It's a feature several people have asked for, but I guess many users don't use public field access and therefor we should treat it more as an advanced "hey by the way, we can also do this" thing.